lib/ed25519-browser.js

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as ed25519 from '@stablelib/ed25519';
+import {randomBytes} from '@stablelib/random';
+
+export default {
+  ...ed25519,
+  async generateKeyPair() {
+    return ed25519.generateKeyPair({randomBytes: () => randomBytes(32)});
+  }
+};
```

Comment:
No need to create a new function every time we generate a key pair -- and we don't need the extra library:

```suggestion
/*!
 * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
 */
import * as ed25519 from '@stablelib/ed25519';

// browser MUST provide "crypto.getRandomValues"
const crypto = self && (self.crypto || self.msCrypto);
if(!crypto.getRandomValues) {
  throw new Error('Browser does not provide "crypto.getRandomValues".');
}

export default {
  ...ed25519,
  async generateKeyPair() {
    const seed = new Uint8Array(32);
    crypto.getRandomValues(seed);
    return ed25519.generateKeyPairFromSeed(seed);
  }
};
```

Reply:
ok switched to using `generateKeyPairFromSeed` and removed the import of `@stabelib/random`: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/b3c0b03ea9a1638c11aafd12a8b1a2e499533945